easybuild/easyconfigs/p/PEST++/PEST++-5.0.5-foss-2020a.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	Custom
+#
+# Notes::
+##
+
+easyblock = 'CMakeMake'
+
+name = 'PEST++'
+version = '5.0.5'
+github_account = "usgs"
+
+homepage = 'https://github.com/usgs/pestpp'
+description = """PEST++ is a software suite aimed at supporting
+ complex numerical models in the decision-support context.
+ Much focus has been devoted to supporting environmental models
+ (groundwater, surface water, etc) but these tools are readily
+ applicable to any computer model.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [GITHUB_LOWER_SOURCE]
```

Comment:
```suggestion
source_urls = ['https://github.com/usgs/pestpp/archive']
```

Reply:
Thx @lexming for catch this one. My bad. Now Fixed.